api/actions/createWorkItemForPR.js

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const { getTitleWithOptionalPrefix } = require('./getTitleWithOptionalPrefix');
+
+const { updateIssue } = require('./updateIssue');
+
+const { formatToGus } = require('./formatToGus');
+
+const GithubEvents = require('../modules/GithubEvents');
+const Github = require('../services/Github');
+const Gus = require('../services/Gus');
+
+module.exports = {
+    eventName: GithubEvents.events.PULL_REQUEST_LABELED,
+    fn: async function(req) {
```

Comment:
This function needs some refactoring. Shares bulk of logic with code that creates work items from Issues. I'll work on extracting into a util function next.